matchmaker.js

```diff
@@ -90,16 +76,16 @@ class Matchmaker {
 
         var success = false
 
-        if (this.match_queues[desired_match_type]["has_player_waiting"] == false) {
-            match_queues[desired_match_type]["has_player_waiting"] = true
-            match_queues[desired_match_type]["waiting_player"] = joining_player
-            match_queues[desired_match_type]["waiting_json_data"] = json_data
-            match_queues[desired_match_type]["waiting_version"] = player_join_version
+        if (this.match_queues[desired_queue_type]["has_player_waiting"] == false) {
```

Comment:
were you taking out "has_player_waiting" and replacing it with just waiting_player?